README.md

```diff
@@ -197,6 +197,40 @@ before the type being defined, for example to define recursive types or as
 the type of `self` inside of classes. In fact, there is no check at all to make sure
 that anotations refer to existing types.
 
+For builtin `list[T]` the following operations are typechecked:
```

Comment:
actually, for lists all operations are typechecked

Reply:
Not quite. E.g. `pop` is not supported. This code does not raise an exception:
```python
def foo(bad: list[str]) -> None:
    bad.pop()

foo([1])
```
The wrapper simply does not check for this.
Support should be added when unifying `list` / `dict` / `set` #46